### DISCUSSION TITLE
cd: not a directory: /opt/homebrew/bin/brew

### DISCUSSION THREAD

**Discussion by asakew** [Homebrew/discussions#5359] [Getting started]
### Output of `brew config`

```shell
not working
```

### Output of `brew doctor`

```shell
not working
```

### Description of issue

I downloaded it and it doesn't work
1: macOS, try our new .pkg installer.
1.2: Command Line Tools for Xcode 15.3: https://developer.apple.com/download/all/?q=Command%20Line%20Tools
2: Git
3: NodeJS

all result screenshots
https://photos.app.goo.gl/2McUCZ3WKVVDCLHD9

**Comment by SMillerDev** [comment]
Not going to open random links, please post the output of your commands as text

**Comment by asakew** [comment]
<img width="678" alt="Снимок экрана 2024-05-12 в 14 42 50" src="https://github.com/Homebrew/discussions/assets/46191008/7660e7ec-d4d3-4d32-9c75-2be900e0d1e0">

Ничья не работает

**Comment by asakew** [comment]
<img width="676" alt="Снимок экрана 2024-05-12 в 14 37 00" src="https://github.com/Homebrew/discussions/assets/46191008/a1aa6693-b83b-4aec-b774-1fc9797e862f">
<img width="783" alt="Снимок экрана 2024-05-12 в 14 44 37" src="https://github.com/Homebrew/discussions/assets/46191008/87af4369-c720-4214-8ee2-ca656cc9bc01">

**Reply by asakew** [reply]
<img width="692" alt="Снимок экрана 2024-05-12 в 17 54 43" src="https://github.com/Homebrew/discussions/assets/46191008/c8361da2-3ccc-4501-96a4-cbbdceb0b2ce">
<img width="692" alt="Снимок экрана 2024-05-12 в 17 50 06" src="https://github.com/Homebrew/discussions/assets/46191008/b330734a-7c7c-44ec-8469-4f886ce4b7a6">

**Comment by asakew** [comment]
<img width="976" alt="Снимок экрана 2024-05-12 в 18 30 42" src="https://github.com/Homebrew/discussions/assets/46191008/2b4871c1-4ce4-4a69-b874-130b369c7a71">

https://github.com/orgs/Homebrew/discussions/775#discussioncomment-9403581

thanks it's working now ❤️ thanks for your help

<img width="1440" alt="Снимок экрана 2024-05-12 в 18 27 09" src="https://github.com/Homebrew/discussions/assets/46191008/6d0e44b5-0772-4c2b-8f28-5ebc7d73f426">

**Comment by asakew** [comment]
❤️ thanks for your help
